### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Yeliseiev**
Search bars added and styled and readme.md edited.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+}
+
+.small_search::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
```
Comment (Andr1yK) [inline src/style.css:63]: this is a default value

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  font-weight: 400;
```
Comment (Andr1yK) [inline src/style.css:64]: this is a default value

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+.search:focus {
+  outline: none;
+  border: 1px solid #01ff5e;
+}
+
+.big_search::placeholder {
+  padding-left: 61px;
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+  font-weight: normal;
```
Comment (Andr1yK) [inline src/style.css:41]: this is a default value

**Review by Andr1yK — APPROVED** [review]

File: src/style.css (hunk 3 above)
Comment (Yeliseiev) [reply src/style.css]: Removed font-family to body

**Pull request closed without merging**